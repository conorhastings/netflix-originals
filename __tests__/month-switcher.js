import React from 'react';
import renderer from 'react-test-renderer';
import MonthSwitcher from '../src/components/month-switcher.js';


test('MonthSwitcher component renders without error', () => {
  const tree = renderer.create(
  	<MonthSwitcher 
  	  month={"August"}
  	  year={"2017"}
  	  changeMonth={() => {}}
  	/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});