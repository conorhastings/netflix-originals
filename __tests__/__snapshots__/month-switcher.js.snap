// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthSwitcher component renders without error 1`] = `
<div
  className="css-1e1pm4a"
>
  <div
    className="css-en2qbu"
    onClick={[Function]}
  >
    ←
  </div>
  <div>
    August
     
    2017
  </div>
  <div
    className="css-en2qbu"
    onClick={[Function]}
  >
    →
  </div>
</div>
`;
